### ISSUE TITLE
Is there a limit of 100 open issues?

### ISSUE THREAD

**Issue by danbri** [pocketworks/git2pdf#4]
(Not that any sane project should be trying to print out 175 open issues, oh no... :)

I looked in the code for 100, saw a curl to  https://api.github.com/repos/pocketworks/repo/issues?per_page=100  ...

**Comment by Arkkimaagi** [comment]
Urgh, I have 104 issues :hankey: 

I'd love to have some option to print the remaining 4 issues or to include them somehow.

**Comment by Splendorr** [comment]
Would also love to print ALL open issues. The whole point of this is to print them out and organize (and hopefully condense/remove a bunch...) my open issues. We make issues for small things. There are 375 open issues in our project right now. I'm having the darnedest time getting a good look at all of them...

**Comment by danbri** [comment]
Yeah, those of us most in need of this tool will get least value from it. Maybe it's a github API issue?

**Comment by stalane** [comment]
Bump

**Comment by vdauwera** [comment]
bump

**Comment by danbri** [comment]
@jgrau 's fork seems most active lately. @jgrau - any insight on this? Is it a Github API limitation? (i.e. we'd need code that paged through the results in several fetches...)

**Comment by jgrau** [comment]
Sorry about the late reply - holiday mode here. I haven't had the issue but I just searched the code and the github api documentation and it seems the default page size for an api result is 30. That number can be changed with the `per_page` parameter and be set to a number _up to 100_. See https://developer.github.com/v3/#pagination. In the code, `per_page` is set to 200. I think the correct approach is to change the code to set `per_page` to 100 and to page through multiple pages if there are more than 100 results.

**Comment by danbri** [comment]
Trying @RichardWallis's [fork](https://github.com/RichardWallis/git2pdf) from the patch #10 he just mentioned, it works for me:

> 19:28 $ ./bin/git2pdf gen schemaorg/schemaorg -u danbri
> Creating a PDF of GitHub issues for repos: schemaorg/schemaorg
> Password: 
> Loading: 100 200 300 400 457 issues
> 457 cards output to issues.pdf

Now to remember how to best print them in OSX Preview. Something like "Print > [set to landscape]; then Layout > Pages per Sheet = 9" then "save as PDF" to capture the compressed layout. I'll test it tomorrow. Anyway this looks good, cheers @RichardWallis :)
